### PR TITLE
It's all about ME!

### PR DESCRIPTION
**Pull request by askpatrickw**
State Motto: "I Lead".

### CODE REVIEW

File: states/maine/elections.toml [hunk 1]
```diff
@@ -1,9 +1,47 @@
 [20200714]
 date = 2020-07-14
 name = "Congressional Primary"
-original_date = 2020-07-14
+original_date = 2020-06-19
+
+[20200714.register]
+# received_by = ?? # Not specified
```
Comment (tannewt) [inline states/maine/elections.toml]: Maybe received_by 21 days before? `There is no cut-off date for registering to vote in person at your town office or city hall. If you want to register to vote by mail or through a voter registration drive, the cut-off date is the close of business on the 21st day before the election.`

**Review by tannewt — CHANGES_REQUESTED** [review]
One note about registration by mail deadline.

File: states/maine/elections.toml (hunk 1 above)
Comment (askpatrickw) [reply states/maine/elections.toml]: Bingo ! thanks.

**Review by tannewt — APPROVED** [review]
Great! Thank you!

**Pull request merged**